backend/src/packages/meditation/libs/enums/meditations-relation.enum.ts

```diff
@@ -0,0 +1,6 @@
+const MeditationRelation = {
```

Comment:
this should be different relations for meditation_entries and meditation_topics

Reply:
Done